test_elasticsearch/test_async/test_transport.py

```diff
@@ -580,8 +582,10 @@ async def test_sniffing_disabled_on_elastic_cloud(self, kwargs):
             == "Sniffing should not be enabled when connecting to Elastic Cloud"
         )
 
-    async def test_sniff_on_start_close_unlocks_async_calls(self):
-        event_loop = asyncio.get_running_loop()
+    async def test_sniff_on_start_close_unlocks_async_calls(self, anyio_backend):
+        if anyio_backend == "trio":
+            pytest.skip("trio does not support sniffing")
```

Comment:
I think it isn't necessary to skip here, or even use the `anyio` fixture and functions, since this whole class is marked with `asyncio` and not `anyio`.